awx_collection/plugins/modules/project.py

```diff
@@ -169,16 +169,10 @@
       required: False
       default: 2
       type: float
-    signature_validation:
```

Comment:
I think line 390 needs to be dropped now too.

Reply:
I believe this is resolved as well now!